.github/workflows/app-build-docker-image.yml

```diff
@@ -91,7 +122,9 @@ jobs:
             APP_VERSION=${{ inputs.application-version }}
             BUILD_SHA=${{ github.sha }}
             ${{ inputs.build-args }}
-          tags: ${{ needs.determine-image-name.outputs.docker-image-name-with-tag }}
+          tags: |
+            ${{ needs.determine-image-name.outputs.gchr-docker-image-name-with-tag }}
+            ${{ needs.determine-image-name.outputs.dockerhub-image-name-with-tag }}
```

Comment:
One other question related to this, what happens if you have the tag that is expected to go to Docker Hub, but the login to Docker Hub didn't happen, which is what is expected for all but prod crawler deploy? Does it just push to ghcr.io in that case?